CONTRIBUTIONS.md

```diff
@@ -0,0 +1,88 @@
+  - it's important that video content displayed on our testnet looks good, and if there are any issues with incorrect metadata, license, playback or associated media these should be reported to the Content Curators
+- original content uploads [indicative score: 500->3000 per upload, depending on quality]
+  - one of the simplest to add value to the network is by adding your own content to our content directory, as a result we will generously reward such contributions.
+- uploading tutorial videos [indicative score: 450->2500 per tutorial, depending on quality]
+  - tutorial videos are an excellent medium for helping people to get to grips with participating on the platform while also augmenting our growing library of content.
+- uploading unique content to the platform [indicative score: 150->700 per upload, depending on uniqueness]
+  - content uploaded doesn't all have to be strictly original, and public domain and creative commons-licensed uploads will also be eligible rewards if they are unique or interesting
+  - duplicate and low quality uploads will not qualify for rewards and content which is improperly licensed or breaks our Terms of Service may result in further penalties
+- creating channels [indicative score: variable]
+  - as part of the above, organizing content uploaded into distinct channels with specific themes or content types will increase rewards
+
+Coding [indicative score: 7500+]:
+- developing tools outside the scope of official bounties (such as wallets, faucet, Telegram Bots, block explorers etc.), optionally through requesting funding through a Council spending proposal represent very valuable contributions to the project and will be rewarded generously with Founding Member points.
+- completing coding bounties will also be rewarded (see `Regular Contributions` below)
+
+Creative/Design [indicative score: 200+ per asset depending on quality]:
+- designing media assets including illustations, gifs, memes, infographics and telegram stickers to be used by Jsgenesis and other project participants will be rewarded generously with leaderboard points.
+
+Security:
+- identifying bugs [indicative score: low severity: 1000, medium severity: 5000, high severity: 25000+]
+  - it is of critical importance that security issues in Joystream are identified and patched
+  - participants who draw attention to or write detailed vulnerability reports about issues with our testnet will therefore be rewarded.
+
+Testing:
+- feedback/suggestions on joystream products [indicative score: 500]
+  - users who provide feedback on Joystream applications and offer suggestions for improvements (either through GitHub issues or on the platform forum) will receive leaderboard points
+- detailed testing of joystream products [indicative score: 2500+ depending on scope]
+  - those who want to conduct formal QA testing are likely to be rewarded to an even greater degree than the above
+
+Regular Contributions [indicative score: variable!]:
+- Bounties
```

Comment:
The score will be proportional to the reward, but not sure what the reward to score should be...